test/built-ins/Array/prototype/copyWithin/fill-holes.js

```diff
@@ -0,0 +1,15 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 22.1.3.3
+description: >
+  Loop from each property, even empty holes.
+includes: [compareArray.js]
+---*/
+
+assert(
+  compareArray(
+    [0, 1, , , 1].copyWithin(0, 1, 4),
+    [1, , , , 1]
```

Comment:
[The `compareArray` helper does not take holes into account](https://github.com/tc39/test262/blob/258d2124ab44d210cb92e3211fd321337fba9525/harness/compareArray.js). This means this test isn't as specific as it ought to be because the following would *also* pass (incorrectly):

  ```js
  assert(
    compareArray(
      [0, 1, , , 1].copyWithin(0, 1, 4),
      [1, undefined, undefined, undefined, 1]
    )
  );
  ```

Better to explicitly check each element (and the array length) in this case.